documentation/shared/attributes.adoc

```diff
@@ -84,7 +84,7 @@
 :K8sDockerSecret: link:https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials[Create a Secret based on existing Docker credentials^]
 :MavenCentral: https://mvnrepository.com/repos/central[Maven Central repository^]
 :Minikube: link:https://kubernetes.io/docs/tasks/tools/install-minikube/[Install and start Minikube]
-:NginxIngressController: link:https://github.com/kubernetes/ingress-nginx[NGINX Ingress Controller for Kubernetes^]
+:NginxIngressController: link:https://github.com/kubernetes/ingress-nginx[NGINX Ingress Controller^]
```

Comment:
The official name is `Ingress-NGINX Controller for Kubernetes` Maybe you can use that, or call it the `Kubernetes NGINX Ingress controller` for example. But again, this is useful to differentiate which one we talk here about.